### ISSUE TITLE
Add Accessibility guidelines to vf.io

### ISSUE THREAD

**Issue by barrymcgee** [vanilla-framework/vanilla-framework#413]
Accessibility guidelines should be added to a top level page within the docs site.

Related: I found this resource which may be informative: http://accessibility.voxmedia.com/

**Comment by yaili** [comment]
Moving to https://github.com/canonical-websites/vanillaframework.io/issues/6
